webapp/src/app/components/dialogs/map-settings/map-content-settings/map-content-settings.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-map-content-settings',
+  templateUrl: './map-content-settings.component.html',
+  styleUrls: ['./map-content-settings.component.scss', '../map-settings.component.scss']
+})
+export class MapContentSettingsComponent implements OnInit {
```

Comment:
Add types to properties/methods. settings is from Type CrossCodeMap. Eventemitter emits a prop, value pair etc.